api/controllers/auth.js

```diff
@@ -46,60 +50,33 @@ const registration = async (req, res) => {
   res.status(201).send(response);
 };
 
-const reset = async (req, res) => {
+const forgotPassword = async (req, res) => {
   const { email } = req.body;
   try {
-    const validation = await emailSchema.validate({ email });
-    const answer =
-      validation && (await services.resetPasswordRequest({ email }));
-    const status = answer.status ? answer.status : 200;
-    const response = answer.messageId ? 'Mail was sent' : answer.message;
-    res.status(status).send(response);
-  } catch (e) {
-    throw new errors.ResetPasswordError(e.message);
-  }
-};
-
-const resetApprovementPassword = async (req, res) => {
-  try {
-    const { linkId } = req.params;
-
-    if (!linkId) {
-      throw new errors.ResetPasswordApproveError('Empty params');
-    }
-
-    const userId = await services.resetPasswordApprove({
-      linkId: decodeURIComponent(linkId),
-    });
-
-    if (userId) {
-      res.status(200).send({
-        status: 200,
-        message: { content: 'Enter New Password', userId },
-      });
-    }
+    await emailSchema.validate(email);
+    const dataEmailing = await services.forgotPassword({ email });
+    const response = BaseResponse.responseBuilder({message: 'Mail sent', data: dataEmailing});
+    res.status(200).send(response);
   } catch (e) {
     throw new errors.ResetPasswordError(e.message);
   }
 };
 
 const resetPassword = async (req, res) => {
-  const { linkId, password, passwordConfirm } = req.body;
+  const { password, confirmPassword } = req.body;
+  const token = req.headers.authorization;
 
-  if (!linkId || !password || !passwordConfirm) {
+  if ( !password || !confirmPassword ) {
```

Comment:
You don't need this code because you have validation below

Reply:
Done. It was for the previous version of validation.